### PR TITLE
Update Tim Chen's entry

### PR DESCRIPTION
**Pull request by cty288**
I updated my previous submission (#104 ). After using `torch.compile()` and `torch.set_float32_matmul_precision("high")`, I was able to train the model faster so a total of 18750 steps could be run in 45 minutes. The best validation loss (3.335) occurred at step 18300. Other model specs kept unchanged. Link: https://wandb.ai/chentim-sh-stanford-university/cs336-assignment1/runs/eyp1ws4o/panel/kwlfruabc?nw=nwuserchentimsh

### CODE REVIEW

**Comment by marcelroed** [comment]
Link gives me a 404
